<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>简单ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnInit">
        <property name="text">
         <string>初始化列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnClearItem">
        <property name="text">
         <string>清空列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkEditable">
        <property name="text">
         <string>可编辑</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QComboBox" name="comboItems">
        <item>
         <property name="text">
          <string>南昌</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/images/image1.png</normaloff>:/icons/images/image1.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>杭州</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icons/images/image5.png</normaloff>:/icons/images/image5.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>有用户数据的ComboBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnCityInit">
        <property name="text">
         <string>初始化城市+区号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboCity">
        <property name="placeholderText">
         <string>请选择一个城市</string>
        </property>
        <property name="duplicatesEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>信息显示</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空文本框</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
